internal/policy/container/has_modified_files.go

```diff
@@ -208,6 +208,22 @@ func (p *HasModifiedFilesCheck) validate(ctx context.Context, layerIDs []string,
 			previousPackage := packageFiles[layerIDs[idx-1]].LayerPackages[previousPackageVersion]
 			currentPackage := ref.LayerPackages[currentPackageVersion]
 
+			var architecture string
+			switch currentPackage.Arch {
```

Comment:
or wrap the below code in an `if architecutre == ""` and don't execute that block? I'm not sure really what we are going for here, or all permutations we need to cover.

Reply:
I went with a different direction for this, and just wrapped the comparison logic in an `if ok` block since we are looking up in a map for the opposite arch in the map.

Reply:
Going a different direction, and trying to exclude these file(s) before we get to the validation logic.